funky_time.py

```diff
@@ -16,18 +16,26 @@ def years_ago(years: int, from_date: datetime = None):
         return from_date.replace(month=2, day=28, year=from_date.year - years)
 
 
-# using years_ago, is this person 18 years old, or older, right now?
-def is_18(birth_date: datetime = None):
+# is the `birth_date` `greater` than `age` now?
+def is_age(age: int, greater: bool, birth_date: datetime = None):
     if isinstance(birth_date, datetime):
-        return years_ago(18, datetime.now()) > birth_date
+        if greater:
+            return years_ago(age, datetime.now()) >= birth_date
+        else:
+            return years_ago(age, datetime.now()) <= birth_date
         # Would be a one line function if it weren't for you meddling kids and your error checking
-        # (JDG) Used > just for clarity. Could arguably be >=
     else:
         raise TypeError("I need a date. Desperately.")  # (JDG) Ask a silly question, get a silly answer.
 
 
+
 def date_to_datetime(date):
     return datetime.combine(date, datetime.min.time())
 
 
-# print(date_to_datetime(datetime.strptime('Jul 18 2001', '%b %d %Y')))
\ No newline at end of file
+# date_to_datetime(datetime.strptime('Jul 18 1001', '%b %d %Y'))
+
+
+
+print(is_age(18, False, date_to_datetime(datetime.strptime('Jul 18 2011', '%b %d %Y'))))
```

Comment:
Before this goes into a non-draft PR could you remove this debug code? :)

Reply:
I'm forever leaving a mess behind me, aren't I!? Soz! That's gone now.